### PR TITLE
BaSP-MR-56: Header components

### PR DESCRIPTION
**Pull request by ukiweskamp**

### CODE REVIEW

**Comment by ukiweskamp** [comment]
[Components - Header](https://trello.com/c/SWgURgIN/56-components-header)

File: src/Components/Activities/index.jsx [hunk 1]
```diff
@@ -146,6 +146,7 @@ function Activities() {
           {showAdd && (
             <form className={styles.form} onSubmit={onSubmit}>
               <input
+                className={styles.input}
```
Comment (Bigzufi) [inline src/Components/Activities/index.jsx:149]: you shold discard this change

**Comment by Bigzufi** [comment]
Looks good to me!

**Review by Bigzufi — APPROVED** [review]

File: src/Components/Shared/Header/Index.jsx
Comment (DelfiLT) [inline src/Components/Shared/Header/Index.jsx:1]: change file name to index

**Review by DelfiLT — APPROVED** [review]

**Comment by ukiweskamp** [comment]
[Components - Header](https://trello.com/c/SWgURgIN/56-components-header)

**Comment by ukiweskamp** [comment]
[Components - Header](https://trello.com/c/SWgURgIN/56-components-header)

**Pull request merged**